bool setupStart
bool gameStart
bool teamGame
bool paused
bool aiGame
bool simGame
int32 seconds
